docs/source-2.0/guides/smithy-cli/index.rst

```diff
@@ -0,0 +1,20 @@
+.. _smithy-cli:
+
+==============
+The Smithy CLI
+==============
+
+The Smithy CLI enables you to quickly iterate on your Smithy models - we’ve listened to community feedback
```

Comment:
I don't think we need to talk about the gradle plugin here. The Gradle plugin will still be useful to lots of people.